x-pack/plugins/security_solution/cypress/integration/timelines/open_timeline.spec.ts

```diff
@@ -75,19 +74,19 @@ describe('Open timeline', () => {
     });
 
     it('should display timeline info - description', () => {
-      cy.get(TIMELINES_DESCRIPTION).first().should('have.text', getTimeline().description);
+      cy.get(TIMELINES_DESCRIPTION).last().should('have.text', getTimeline().description);
```

Comment:
Why `last` instead of `first`?

Reply:
The initial selector was the incorrect one. In this test we first navigate to the timelines page and then we open the timeline. Doing that, we have the selectors repeated (x2) because the timeline is opened over the timelines page. The first one refers to the field on the timelines page and the last one on the timeline. As the values are the same, the tests were not failing.